crates/subspace-wasm-tools/src/lib.rs

```diff
@@ -83,3 +85,21 @@ pub fn create_runtime_bundle_inclusion_file(
         panic!("Must be able to write to {target_file_name}: {error}");
     });
 }
+
+/// Read the core domain runtime blob file from the system domain runtime blob file.
+///
+/// The `section_contents_name` should be driven from `DomainId::link_section_name`.
+pub fn read_core_domain_runtime_blob(
+    system_domain_bundle: &[u8],
+    section_contents_name: String,
```

Comment:
Why change `core_domain_id` to `section_contents_name`?

Reply:
I try to reduce one dependency for `subspace-wasm-tools`...

Reply:
I don't see why that matters, the previous version is strongly typed and more descriptive, the API itself can almost tell everything it does: read the core domain runtime blob from the system domain bundle with the given core domain id.